src/Components/ExpressionInput/FunctionExpressionInput/FunctionExpressionInput.purs

```diff
@@ -87,6 +87,11 @@ render state =
                   , onEnterPressActionEvent UpdateExpression
                   , className "form-control"
                   ]
+              , HH.a
+                  [ className "btn btn-info"
+                  , HP.href "https://github.com/michalkonecny/robust-plots/blob/master/docs/syntax.md"
```

Comment:
I think it is best if this link would open a new tab/window by default.  It should not replace the plotter app.  The same also in the the parametric expression input.

Reply:
Resolved in latest commit